src/pythainer/builders/__init__.py

```diff
@@ -375,7 +387,7 @@ def build(self, dockerfile_savepath: PathType = "") -> None:
 
             command = self.get_build_commands(
                 dockerfile_path=dockerfile_path,
-                docker_build_dir=temp_path,
+                docker_build_dir=Path(os.path.abspath(docker_context)) if docker_context else temp_path,
```

Comment:
I prefer to use pathlib
```suggestion
                docker_build_dir=Path(docker_context).resolve() if docker_context else temp_path,
```